packages/clerk-js/src/ui/contexts/components/SignUp.ts

```diff
@@ -65,8 +65,8 @@ export const useSignUpContext = (): SignUpContextType => {
     options,
     {
       ...ctx,
```

Comment:
doesn't this mean that `SignInContext` and `SignUpContext` still contain a `forceRedirectUrl` prop? Can we completely remove any and all usage of `forceRedirectUrl` except at the component boundary?

Reply:
I'll remove from the context, as far as I can tell `forceRedirectUrl` isn't actually used outside of this assignment in the components, so we _should_ be safe.